scene/animation/animation_blend_space_1d.cpp

```diff
@@ -30,12 +30,22 @@
 
 #include "animation_blend_space_1d.h"
 
+#include "animation_blend_tree.h"
+
 void AnimationNodeBlendSpace1D::get_parameter_list(List<PropertyInfo> *r_list) const {
 	r_list->push_back(PropertyInfo(Variant::FLOAT, blend_position));
+	r_list->push_back(PropertyInfo(Variant::INT, closest, PROPERTY_HINT_NONE, "", PROPERTY_USAGE_NONE));
+	r_list->push_back(PropertyInfo(Variant::FLOAT, length_internal, PROPERTY_HINT_NONE, "", PROPERTY_USAGE_NONE));
 }
 
 Variant AnimationNodeBlendSpace1D::get_parameter_default_value(const StringName &p_parameter) const {
-	return 0;
+	if (p_parameter == closest) {
+		return -1;
+	} else if (p_parameter == length_internal) {
+		return 0;
+	} else {
+		return 0;
+	}
```

Comment:
```suggestion
	if (p_parameter == closest) {
		return -1;
	} else {
		return 0;
	}
```
It can be written like above.